api/v1alpha1/storageconsumer_types.go

```diff
@@ -62,11 +79,15 @@ type StorageConsumerStatus struct {
 	// State describes the state of StorageConsumer
 	State StorageConsumerState `json:"state,omitempty"`
 	// CephResources provide details of created ceph resources required for external storage
+	// +kubebuilder:deprecatedversion:warning="This field has been deprecated and will be removed in future."
 	CephResources []*CephResourcesSpec `json:"cephResources,omitempty"`
 	// Timestamp of last heartbeat received from consumer
 	LastHeartbeat metav1.Time `json:"lastHeartbeat,omitempty"`
 	// Information of storage client received from consumer
-	Client ClientStatus `json:"client,omitempty"`
+	Client                ClientStatus          `json:"client,omitempty"`
+	RadosNamespace        RadosNamespaceStatus  `json:"radosNamespace,omitempty"`
+	CephFsSubVolumeGroup  SubVolumeGroupStatus  `json:"cephFsSubVolumeGroup,omitempty"`
+	CephCsiClientProfiles []ClientProfileStatus `json:"cephCsiClientProfile,omitempty"`
```

Comment:
There is no point in having this here anymore, what we want is to reflect the configmap we are using

Reply:
removed.